service/build.gradle

```diff
@@ -9,7 +9,16 @@ plugins {
 apply from: 'generators.gradle'
 apply from: 'publishing.gradle'
 
+repositories {
```

Comment:
Could we add the repo here instead if it's not currently included? https://github.com/DataBiosphere/terra-billing-profile-manager/blob/main/buildSrc/src/main/groovy/bio.terra.profile.java-common-conventions.gradle#L25